### PR TITLE
Gateway - don't log updates on startup

### PR DESCRIPTION
**Pull request by trevor-scheer**
The gateway currently spits out INFO logs about updating schema on both startup and during schema updates. It really should only log "updating schema" if a previous schema exists.

** Also removes extraneous neighboring else block (the same is resolved immediately below)

### CODE REVIEW

File: packages/apollo-gateway/src/index.ts [hunk 1]
```diff
@@ -285,15 +285,16 @@ export class ApolloGateway implements GraphQLService {
     ) {
       this.logger.debug('No change in service definitions since last check');
       return;
-    } else {
-      this.serviceDefinitions = result.serviceDefinitions;
+    }
+
+    if (previousSchema) {
```
Comment (zionts) [inline packages/apollo-gateway/src/index.ts]: might we want a log line for like "started up w/ schema from service w/ ID `blah`" ?

File: packages/apollo-gateway/src/index.ts [hunk 2]
```diff
@@ -236,6 +236,8 @@ export class ApolloGateway implements GraphQLService {
 
   public async load(options?: { engine?: GraphQLServiceEngineConfig }) {
     await this.updateComposition(options);
+    const graphId = options && options.engine && options.engine.graphId;
+    this.logger.info(`Gateway loaded schema${graphId && ` for service ${graphId}`}.`)
```
Comment (zionts) [inline packages/apollo-gateway/src/index.ts]: probably want a variant in there? And maybe the method it used! I've often been uncertain from my logs if I'm running in managed or unmanaged mode :)

Comment (trevor-scheer) [reply packages/apollo-gateway/src/index.ts]: 👍 resolved in eded62a

File: packages/apollo-gateway/src/index.ts (hunk 1 above)
Comment (trevor-scheer) [reply packages/apollo-gateway/src/index.ts]: Resolved in 91a4e89

**Pull request merged**